### ISSUE TITLE
Custom module error

### ISSUE THREAD

**Issue by Falcort** [directus/directus#3903]
Hi,

I am trying to create a custom module by following the documentation [here](https://docs.directus.io/guides/extensions/modules.html) (I barely or even did not change anything from the doc)

But the module isn't appearing and I am also getting a related console warning : 
```
Couldn't load custom module "test"
TypeError: undefined is not an object (evaluating 'e.default.routes.map')
```

I also encountered an issue that might be related, when doing the command : 
```
npm i -D rollup rollup-plugin-commonjs rollup-plugin-node-resolve rollup-plugin-terser rollup-plugin-vue@5.0.0 @vue/compiler-sfc vue-template-compiler
```
 and then you execute : 
```
npx rollup -c
```
you get the following error output : 
```
[!] Error: 

Vue packages version mismatch:

- vue@3.0.5 (/Users/falcort/Desktop/my-project/bkp/node_modules/vue/index.js)
- vue-template-compiler@2.6.12 (/Users/falcort/Desktop/my-project/bkp/node_modules/vue-template-compiler/package.json)

This may cause things to work incorrectly. Make sure to use the same version for both.
If you are using vue-loader@>=10.0, simply update vue-template-compiler.
If you are using vue-loader@<10.0 or vueify, re-installing vue-loader/vueify should bump vue-template-compiler to the latest.
```
But if you use YARN for the installation then the problem is solved and the rollup execute without problems.

The expected result would be my test component to be working and accessible.

My environment : 
Database : Postgres 12
Directus : 9.0.0-rc.34
NodeJS : v15.6.0
Web : Safari 14.0.2

**Comment by fljs** [comment]
workaround found in discord:

Got it to work by installing npm install rollup-plugin-vue@next and removing the vue-template-compiler.

**Comment by nickrum** [comment]
This should be fixed by #5065
